src/Quote.tsx

```diff
@@ -0,0 +1,78 @@
+import React, { memo, forwardRef, ReactNode } from "react";
+import { symToStr } from "tsafe/symToStr";
+import { assert } from "tsafe/assert";
+import type { Equals } from "tsafe";
+import { fr } from "./lib";
+import { cx } from "./lib/tools/cx";
+
+// We make users import dsfr.css, so we don't need to import the scoped CSS
+// but in the future if we have a complete component coverage it
+// we could stop requiring users to import the hole CSS and only import on a
+// per component basis.
```

Comment:
This comment don't need to be reported everywhere.  
Infact, I am questioning if it makes sence to include css that, as of today is useless. YDYT?

Reply:
yeah, one issue one that topic could replace those comments